src/Sylius/Bundle/CoreBundle/Form/Extension/AddressTypeExtension.php

```diff
@@ -43,7 +43,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $countryCodeField = $builder->create(
             $oldCountryCodeField->getName(),
             $oldCountryCodeField->getType()->getInnerType()::class,
-            array_replace($oldCountryCodeField->getOptions(), ['choices' => $channel->getCountries()->toArray()]),
+            array_replace($oldCountryCodeField->getOptions(), ['choices' => $channel->getEnabledCountries()->toArray()]),
```

Comment:
There is if clause above this snippet, which most likely should be updated as well.

```patch
---        if ($channel === null || $channel->getCountries()->count() === 0) {
+++        if ($channel === null || $channel->getEnabledCountries()->count() === 0) {
```